src/commands/QuotesCommand/utils.js

```diff
@@ -0,0 +1,27 @@
+const fs = require('fs');
+
+const writeToFile = (filePath, data) => {
+    fs.writeFile(filePath, JSON.stringify(data, null, 1), (err) => {
+        if(err){
+            console.error(err);
+            return;
+        }
+    });
+    return;
+}
+
+const readQuotesFromFile = (filePath) => {
+    fs.readFile(filePath, { flag: "a+" }, (err, contents) => {
+        if (err) {
+            console.error(err);
+            return {};
+        }
+        if (!contents[0]) return {};  // file was just created
+        return JSON.parse(contents);
```

Comment:
This return won't work, because you are returning inside the callback and not the function "readQuotesFromFile"

Reply:
You have to figure out a way to make this the return of the method.
Maybe use readFileSync?

Reply:
As it is 'this.quotes' is always 'undefined'

Reply:
You're right, I didn't have too much time to test. Later (or tomorrow) I'll look for an alternative